packages/frontend-plugin-api/api-report.md

```diff
@@ -993,6 +995,20 @@ export { identityApiRef };
 
 export { microsoftAuthApiRef };
 
+// @public
+export class MultipleAnalyticsApi implements AnalyticsApi_2, AnalyticsApi {
```

Comment:
This needs to be in `core-compat-api`, since we can't have a dependency on `core-plugin-api` from here.